.github/workflows/ci.yml

```diff
@@ -84,7 +84,7 @@ jobs:
       run: python -m pip install --upgrade pip tox
 
     - name: Check
-      run: tox -c tox.ini -e ${{ matrix.python.tox }}-${{ matrix.task.tox }}
+      run: tox -c tox.ini -e ${{ matrix.task.tox }}
```

Comment:
do we  need to specify the tox  config file? Looks like we are using the default one.

Reply:
Doesn't seem like we do.  That's left from Travis I assume as I've never done that before.